u01_telemachus.xml

```diff
@@ -205,14 +205,14 @@
 <p><lb n="010203"/>He shook his constraint from him nervously.
 <lb n="010204"/><said who="bm">―And what is death,</said> he asked, <said who="bm">your mother's or yours or my own? You
 <lb n="010205"/>saw only your mother die. I see them pop off every day in the Mater and
-<lb n="010206"/>Richmond and cut up into tripes in the dissectingroom. It's a beastly thing
+<lb n="010206"/>Richmond and cut up into tripes in the <distinct type="nonstandard-compound">dissectingroom.</distinct> It's a beastly thing
 <lb n="010207"/>and nothing else. It simply doesn't matter. You wouldn't kneel down to
 <lb n="010208"/>pray for your mother on her deathbed when she asked you. Why? Because
 <lb n="010209"/>you have the cursed jesuit strain in you, only it's injected the wrong way.
 <lb n="010210"/>To me it's all a mockery and beastly. Her cerebral lobes are not
 <lb n="010211"/>functioning. She calls the doctor sir Peter Teazle and picks buttercups off
 <lb n="010212"/>the quilt. Humour her till it's over. You crossed her last wish in death and
-<lb n="010213"/>yet you sulk with me because I don't whinge like some hired mute from
+<lb n="010213"/>yet you sulk with me because I don't <distinct type="dialect">whinge</distinct> like some hired mute from
```

Comment:
I'm not so sure about this one. "Whinge" seems fairly standard (though informal) British English. But I also feel like erring on the side of more markup is a good idea.